src/metabase/driver/interface.clj

```diff
@@ -1,14 +1,4 @@
-(ns metabase.driver.interface
-  "Protocols that DB drivers implement. Thus, the interface such drivers provide."
-  (:import (clojure.lang Keyword)))
-
-(def ^:const driver-optional-features
-  "A set on optional features (as keywords) that may or may not be supported by individual drivers."
-  #{:foreign-keys
-    :nested-fields                         ; are nested Fields (i.e., Mongo-style nested keys) supported?
-    :set-timezone
-    :standard-deviation-aggregations
-    :unix-timestamp-special-type-fields})
+(ns metabase.driver.interface)
```

Comment:
One thought here is that instead of keeping this namespace as separate, lets just move these things into `metabase.driver` directly.

I think it's nice if the principle interface and top level functions for interacting with drivers are all in the `metabase.driver` namespace for simplicity sake.  It makes that namespace a bit bigger, but in this case I think that's appropriate.  I don't think there is a great deal of value in having this code spread out in its own interface in this particular case.